scully/renderPlugins/puppeteerRenderPlugin.ts

```diff
@@ -9,8 +9,10 @@ import {logError, yellow} from '../utils/log';
 import {launchedBrowser} from './launchedBrowser';
 
 export const puppeteerRender = async (route: HandledRoute): Promise<string> => {
-  const {version} = JSON.parse(readFileSync(join(__dirname, '../package.json')).toString()) || '0.0.0';
-  const path = `http://localhost:${scullyConfig.appPort}${route.route}`;
+  // const {version} = JSON.parse(readFileSync(join('../package.json')).toString()) || '0.0.0';
```

Comment:
@SanderElias we need you to fix this. It was causing an error when we were running it. Trying to read package.json from `/dist/scully/package.json` which obviously doesn't exist.